frame/staking/src/lib.rs

```diff
@@ -766,7 +769,7 @@ impl<T: Trait> SessionInterface<<T as frame_system::Trait>::AccountId> for T whe
 	}
 }
 
-pub mod weight {
+mod weight {
```

Comment:
these are now superfluous, no?

Reply:
This will be removed in the next commit, yes.